scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/controller/JobsController.java

```diff
@@ -626,6 +619,35 @@ public void onSuccess(String jsonString) {
                                                         });
     }
 
+    public void checkJobPermissionMethod(Job job, Label label, KeyValueGrid variablesGrid,
```

Comment:
It would be better that the **checkJobPermissionMethod** returns a boolean !!

Reply:
The controller cannot return a result to the view. It should be a void